framework/doc/content/source/physics/DiffusionCG.md

```diff
@@ -0,0 +1,49 @@
+# DiffusionCG
+
+!syntax description /Physics/DiffusionCG/DiffusionCG
```

Comment:
this is one place where I deviate from an Action because the name of the syntax is the same as the name of the object. Your typical action here would be `/Kernels/AddKernelAction` for example.

Here we have a Physics which does work like an action (and inherits from Action), but the doco is actually meant to be looked at (unlike let's AddBCAction, which is not very interesting in itself and forwards to the BC page) so I want it documented under Physics/its own name. 

So the traditional MOOSE way here would be:
- syntax documentation for Physics, does not show up in source index under physics (unless we symlink it there?)
- instead of calling the Physics object DiffusionCG, it'd be called AddDiffusionCGAction. But this is not great:

- a DiffusionCG is not a MooseObject, unlike each XXX that the AddXXXAction adds usually. So all the code for the DiffusionCG is in the "Action" / Physics, and not in another class.

Reply:
I can't tell if you're looking for feedback here?

Reply:
Always happy to get feedback. I dont know how to do better but maybe you or @dschwen will know a better way

Reply:
I can't think of anything off the top of my head. Hopefully over time we can come up with something better. I'm not too worried about it at the moment